test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval_long_expiration.test

```diff
@@ -72,7 +71,7 @@ payload='<?xml version="1.0"?>
     <attribute>temperature</attribute>
   </attributeList>
   <reference>http://127.0.0.1:'${LISTENER_PORT}'/notify</reference>
-  <duration>P99Y</duration>
+  <duration>PT11S</duration>
```

Comment:
It was the only way I managed to have this test not failing under valgrind.
The timing under valgrind is completely off ...